Description: Fix PHP 8.x deprecation passing null to strpos in phodevi_gpu
Author: Rinat Gumirov <rinat.gumirov66@gmail.com>
Forwarded: not-needed
Last-Update: 2025-11-29

--- phoronix-test-suite-10.8.4.orig/pts-core/objects/phodevi/components/phodevi_gpu.php
+++ phoronix-test-suite-10.8.4/pts-core/objects/phodevi/components/phodevi_gpu.php
@@ -341,7 +341,7 @@ class phodevi_gpu extends phodevi_device
 			{
 				$info = shell_exec('xrandr 2>&1 | grep "*"');
 
-				if(strpos($info, '*') !== false)
+				if(strpos((string) $info, '*') !== false)
 				{
 					$res = pts_strings::trim_explode('x', $info);

